docs/src/main/sphinx/release/release-454.md

```diff
@@ -0,0 +1,88 @@
+# Release 454 (7 Aug 2024)
+
+## General
+
+* Add support for retrieving lightweight query information from the `QueryInfo`
+  endpoint with the new `pruned` property. ({issue}`20843`)
+* Improve performance of queries with multiple distinct aggregations. ({issue}`22355`)
+* Skip query completed notifications in the event listener when there are too
```

Comment:
This should describe the user visible impact and the reason why this matters. It should be something like: "Improve server stability by discarding events when event listeners are slow to process them.

BTW, we'll probably have to change the name of the property, as "concurrent" doesn't convey the right notion. See my comment: https://github.com/trinodb/trino/pull/22911#discussion_r1708076899

cc @sopel39